torchgeo/datasets/utils.py

```diff
@@ -375,3 +378,19 @@ def collate_dict(samples: List[Dict[str, Any]]) -> Dict[str, Any]:
                 sample[key] for sample in samples
             ]  # type: ignore[assignment]
     return collated
+
+
+def rasterio_loader(path: str) -> np.ndarray:  # type: ignore[type-arg]
```

Comment:
Could we import `torchvision`'s `default_loader` and rename it `pil_loader` here? Then we can import it and use it elsewhere without wondering what `default_loader` is.